tests/config.js

```diff
@@ -157,6 +157,18 @@ describe( 'set', () => {
 		expect( config.get( 'date' ) ).to.be.an.instanceof( Date );
 		expect( config.get( 'instance' ) ).to.be.an.instanceof( SomeClass );
 	} );
+
+	it( 'should not transform keys to lowerCase', () => {
```

Comment:
I think this test is not necessary. It could have been useful during development related issue, but keeping it might be confusing.